bin/run-r-scripts

```diff
@@ -10,7 +10,6 @@ echo 'options(repos = c(CRAN = "https://packagemanager.rstudio.com/all/__linux__
 echo 'options(HTTPUserAgent = sprintf("R/%s R (%s)", getRversion(), paste(getRversion(), R.version$platform, R.version$arch, R.version$os)))' >> ~/.Rprofile
 # Rscript -e "update.packages(ask = FALSE)"
 Rscript -e 'pkgs_to_install <- c("assertthat", "bookdown", "config", "conflicted", "countrycode", "data.table", "devtools", "dplyr", "forcats", "fs", "fst", "ggplot2", "glue", "here", "highcharter", "janitor", "jsonlite", "knitr", "purrr", "readr", "readxl", "renv", "reshape2", "rlang", "rmarkdown", "rstudioapi", "scales", "stringr", "testthat", "tibble", "tidyr", "tidyselect", "usethis", "withr", "writexl", "zoo"); install.packages(setdiff(pkgs_to_install, installed.packages()))'
```

Comment:
@cjyetman do we need to install packages? The packages we need might already be installed in the image 2dii/r-packages -- or we should extend it so they are.

Reply:
That line shouldn't be unnecessary. I left it as a fail safe in case the docker image hadn't been updated yet.

Reply:
Good idea. Should I ensure all packages are includes in 2dii/r-packages and then remove this line?

Reply:
I like the sound of that